composer.json

```diff
@@ -27,12 +27,12 @@
     }
   ],
   "require": {
-    "php": ">=5.5",
-    "guzzlehttp/guzzle": "~6.0@dev|>=6",
-    "psr/log": "~1|~1.0|1.0.0|dev-master"
+    "php": "^5.5.9 || ^7.0",
```

Comment:
Yes, I understand it's an older version of PHP. But we came to the conclusion that this should be up to the user, not the library to force users to update PHP. Since this library works perfectly on PHP 5.5.0, there is no reason to block users from installing it.